### PR TITLE
Remove actor-model and make scripts more robust

### PR DESCRIPTION
**Pull request by simoons95**
The actor-model will be moved to the [kata repo](https://code.euranova.eu/enx-docs/community-dev-katas), in order to keep this repo for scaffolds only.

Also, sometimes, `$PWD` contains spaces. Adding `"` around it makes it more robust.

### CODE REVIEW

File: scaffolds/python/src/test/test_my_module.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import pytest
 from my_module import my_function
 
+
 def test_my_function():
     assert my_function(1, 3) == 4
 
+
```
Comment (jbruggem) [inline scaffolds/python/src/test/test_my_module.py:8]: Was this line added on purpose ?

**Review by jbruggem — CHANGES_REQUESTED** [review]
Sorry for the inexcusable delay ! 

Changes look good. One problem I see though: the permissions of the files have been changed, and they are not executable any more IIUC. I think that part should be reverted.

File: scaffolds/python/src/test/test_my_module.py (hunk 1 above)
Comment (simoons95) [reply scaffolds/python/src/test/test_my_module.py:8]: Yes: you should have two blank lines between module functions (and one between class functions)

"Surround top-level function and class definitions with two blank lines." ( https://peps.python.org/pep-0008/#blank-lines )

**Comment by simoons95** [comment]
> Changes look good. One problem I see though: the permissions of the files have been changed, and they are not executable any more IIUC. I think that part should be reverted.

That's not on purpose (my PC is on windows, it may have had an influence?). I don't know how to revert that or how to check if it is corrected.

**Comment by jbruggem** [comment]
@simoons95 I don't know the specifics of how git for windows works. I suggest using WSL to checkout the repo and commit changes: if you're on linux, the permissions of the file on the file system will always match the permissions in the commit.

**Review by jbruggem — APPROVED** [review]
LGTM !

**Pull request merged**